app/services/katello/registration_manager.rb

```diff
@@ -128,6 +128,7 @@ def joined_hostnames(hosts)
       def unregister_host(host, options = {})
         organization_destroy = options.fetch(:organization_destroy, false)
         unregistering = options.fetch(:unregistering, false)
+        reregistering = options.fetch(:reregistering, false)
```

Comment:
Would be good to add `reregistering` to the comment-documentation 4 lines above here